src/Statistics-Discussion.ipynb

```diff
@@ -138,56 +138,58 @@
+    "Dieses Kapitel stellt die Entwicklung der einzelnen KI-Versionen dar, wobei die erste KI-Iteration übergangen wird, da diese nur zufällige Züge auswählt und somit für keine andere KI ein ernsthafter Gegner ist. Im Folgenden ist eine Zusammenfassung von Spielen des Stockfish Spielers gegen alle implementierten KI-Iterationen zu sehen.\n",
     "\n",
-    "Bei einer Depth von drei beträgt die durchschnittliche Zugzeit aller KIs unter eine Minute. Während die `Exercise03AI` mit 51,2 Sekunden die längste Rechendauer vorweist, ist die `Exercise06AI` mit 9,4 Sekunden die schnellste KI.\n",
+    "<img src=\"images/StockfishResultsDepth4.png\" width=\"800px\" />\n",
     "\n",
-    "<img src=\"images/Depth3AvgDuration.png\" width=\"900px\" />\n",
+    "Aus dieser Statistik ist zu entnehmen, dass die `Exercise02AI` das schlechteste Resultat liefert, da deren Bewertung eines Zugs nur über den einfachen Materialwert stattfindet. Alle folgenden KI-Iterationen sollten sich nicht in ihrer Spielstärke unterscheiden, sondern lediglich in ihrer Performanz. Um einen fairen Vergleich sicherzustellen, wird der `max_depth` Parameter der `Exercise08AI`, also die maximale Suchtiefe der Singular Value Extension (SVE), immer der Suchtiefe der anderen KI-Versionen angepasst. Die SVE wird somit nicht effektiv in dieser Diskussion verwendet. Mit Ausnahme der Ergebnisse der `Exercise07AI` wird diese Annahme auch von den Statistiken unterstützt. Diese Ausnahme ist vermutlich auf Messungenauigkeiten zurückzuführen, welche durch die Sortierung der Züge durch das Progressive Deepening auftreten können.\n",
     "\n",
-    "Die Berechnungsdauer wächst exponentiell mit steigender Suchtiefe. Auch bei einer Tiefe von 4 bleibt die sechste KI Version mit 72,9 Sekunden die schnellste KI. Mit 2117,9 Sekunden pro Zug ist `Exercise03AI` erneut mit Abstand die KI mit der längsten Berechnungsdauer. Selbst die finale Version der KI, die durch die Singular-Value-Extension bis zu acht Züge im Voraus berechnet, ist mit 1035,1 Sekunden mehr als doppelt so schnell.\n",
+    "Im folgenden Diagramm wird die maximale Suchtiefe der SVE auf acht gesetzt, die generelle Suchtiefe beträgt weiterhin vier.\n",
     "\n",
-    "<img src=\"images/Depth4AvgDuration.png\" width=\"900px\" />\n",
+    "<img src=\"images/Ex08ResultsDepth4MaxDepth8.png\" width=\"800px\" />\n",
     "\n",
-    "### Exercise08AI\n",
+    "In diesem Diagramm spiegelt das Resultat die erwarteten Werte wider. Die `Exercise08AI` gewinnt aufgrund der SVE gegen ihre Vorgängerversionen oder läuft aufgrund der Zugwiederholung in ein Remis.\n",
```

Comment:
```suggestion
    "In diesem Diagramm spiegelt das Resultat die erwarteten Werte wieder. Die `Exercise08AI` gewinnt aufgrund der SVE gegen ihre Vorgängerversionen oder läuft aufgrund der Zugwiederholung in ein Remis.\n",
```

Reply:
#Wontfix. See [this link](https://www.korrekturen.de/beliebte_fehler/wiederspiegeln.shtml) for more details.